src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -527,6 +527,140 @@ internal sealed partial class LazySimpleAssignmentExpression : BaseSimpleAssignm
     }
 
     /// <summary>
+    /// Represents a deconstruction assignment expression.
+    /// </summary>
+    internal abstract partial class BaseDeconstructionAssignmentExpression : AssignmentExpression, IDeconstructionAssignmentExpression
+    {
+        public BaseDeconstructionAssignmentExpression(SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+            base(OperationKind.DeconstructionAssignmentExpression, semanticModel, syntax, type, constantValue, isImplicit)
+        {
+        }
+        public override IEnumerable<IOperation> Children
```

Comment:
>public override IEnumerable<IOperation> Children [](start = 8, length = 48)

Is there a reason to override this property in derived classes rather than doing this in ```AssignmentExpression```?